### DISCUSSION TITLE
Disable Dependency Dashboard globally

### DISCUSSION THREAD

**Discussion by tobru** [renovatebot/renovate#12162] [Request Help] [answered]
### What would you like Renovate to be able to do?

The dependency dashboard can only be disabled on a per repository setting. I'd like to disable the dependency dashboard for a whole GitHub organization, so that it I don't have to do it for each and every repository individually.

### If you have any ideas on how this should be implemented, please tell us here.

Maybe disable dependency dashboard by default and make it an opt-in feature.

### Is this a feature you are interested in implementing yourself?

No

**Comment by viceice** [comment, marked as answer]
You've already multiple ways to do that:

* use a shared preset and exclude `:dependencyDashboard` preset
* add a `force` obeject to global config ( self-hosted only)


If you're using renovate org widde, you should use your own shared preset, which can extend the `config:base` if you like.
You can also have your own default onboarding config then. See https://github.com/renovatebot/.github for samples

**Comment by viceice** [comment]
Disable by default isn't an option, as it's disabled by default and only enabled via our `config:base` preset. 
That preset is ment as our recommend preset not required, see #12024
